packages/init/src/utils/tracker.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * DUPLICATION
+ * this file is a duplication of ServerTracker in sm-ui
+ * this duplication has been done on purpose to simplify the merge if init becomes part of sm-ui
+ * or if we decide to share the tracker implementation from a lib, but it'll need some refactoring
+ * Duplication means no entanglement so don't merge this with the other implem and keep the structure.
```

Comment:
`if init becomes part of sm-ui` .... was that a discussion i missed out on?

Reply:
It's an information I got from Hugo that made me do it this way since I don't have much info on the next steps. I think that's a discussion you should bring to him

Reply:
@MarcMcIntosh a good amount of code was added to the core because it had to be shared between sm-ui and init. But when we started defining the core more precisely, we realized the init simply is a dependent of sm-ui. To say it differently: the init is the init of sm-ui. This explains why they share common code, so the init could be a script (like start) in sm-ui. No need to rush things as everything works as expected right now